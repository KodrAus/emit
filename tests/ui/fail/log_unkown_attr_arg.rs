#![feature(stmt_expr_attributes, proc_macro_hygiene)]

fn main() {
    tracing_subscriber::fmt().init();

    let v = "Some data";

    emit::info!("Logging with a deeply nested {#[emit::as_display(not_an_arg: false)] value: v}");
}
