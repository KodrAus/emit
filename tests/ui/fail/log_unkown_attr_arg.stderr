error: custom attribute panicked
 --> tests/ui/fail/log_unkown_attr_arg.rs:8:48
  |
8 |     emit::info!("Logging with a deeply nested {#[emit::as_display(not_an_arg: false)] value: v}");
  |                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: unexpected field `not_an_arg`
